Makefile

```diff
@@ -138,9 +138,19 @@ test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $$(go list ./... | grep -v /e2e) -coverprofile cover.out
 
 # Utilize Kind or modify the e2e tests to load the image locally, enabling compatibility with other vendors.
-.PHONY: test-e2e  # Run the e2e tests against a Kind k8s instance that is spun up.
+.PHONY: test-e2e  # Run the e2e tests against a Kind k8s instance that is spun up using emulator mode
 test-e2e:
-	@export IMG_TAG=test-e2e; make docker-build; go test ./test/e2e/ -v -ginkgo.v --timeout 20m
+	@export IMG_TAG=test-e2e EMULATED="true"; make docker-build; go test ./test/e2e/ -v -ginkgo.v --timeout 20m
+
+.PHONY: test-e2e-ocp
+test-e2e-ocp:
+	@export OCP_MODE="true"; make docker-build; go test ./test/e2e/ -v -ginkgo.v --timeout 20m
```

Comment:
Why combine running in emulated mode, building the docker image and then running the tests in one target? How will one run the e2es on an already provisioned OCP cluster with real GPUs ?

Reply:
This runs e2e tests on an OCP cluster and the real GPUs, not an emulator mode. We have one more target `test-e2e-ocp-emulated` to run e2e tests on an OCP cluster in emulator mode